enterprise/server/remote_execution/commandutil/commandutil_unix_test.go

```diff
@@ -120,13 +120,15 @@ func TestRun_Killed_ErrorResult(t *testing.T) {
 		assert.Contains(t, res.Error.Error(), "signal: interrupt")
 		assert.Equal(t, -1, res.ExitCode)
 	}
-	{
-		res := runSh(ctx, "kill -TERM $$")
-
-		require.Error(t, res.Error)
-		assert.Contains(t, res.Error.Error(), "signal: terminated")
-		assert.Equal(t, -1, res.ExitCode)
-	}
+	// TODO(sluongng): go_test binary starts to ignore SIGTERM since https://github.com/bazelbuild/rules_go/pull/3920.
+	// Need to find another way to test this.
+	// {
+	// 	res := runSh(ctx, "kill -TERM $$")
+
+	// 	require.Error(t, res.Error)
+	// 	assert.Contains(t, res.Error.Error(), "signal: terminated")
+	// 	assert.Equal(t, -1, res.ExitCode)
+	// }
```

Comment:
@bduffany FYI.

I thought this test only killed the `sh` shell, but for some reason sigterm is being ignored here.

Reply:
You can restore the original behavior with `signal.Reset(os.SIGTERM)`, but its slightly suspicious that this is needed here.

Reply:
Let me try bisecting rules_go tmr to verify the root cause

Reply:
Bisect confirmed the culprit PR.

`signal.Reset` did not fix the test, but adding our own `signal.Notify` handler worked.